hovers.css

```diff
@@ -0,0 +1,6 @@
+ 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla3/pullRequest?prid=6027752)